meta-lmp-base/recipes-support/compose-apps-early-start/compose-apps-early-start/compose-apps-early-start

```diff
@@ -41,14 +41,29 @@ start_restorable_apps() {
 
 	if [ -f $default_apps_path ] ; then
 		for app in `cat $default_apps_path`; do
-			shortlist="${shortlist}${app},"
+			if [ -e "/var/sota/compose-apps/${app}" ] ; then
+				start_compose_app ${app}
+			else
+				shortlist="${shortlist}${app},"
+			fi
 		done
 		if [ ! -z $shortlist ] ; then
 			shortlist=${shortlist::-1}
                 fi
 	fi
 
-	aklite-apps run --apps "${shortlist}"
+	if [ -z "${shortlist}" ] ; then
+		all=`ls /var/sota/compose-apps 2> /dev/null`
+		if [ -n "${all}" ] ; then
+			for app in ${all} ; do
+				start_compose_app ${app}
+			done
+		else
+			aklite-apps run --apps "${shortlist}"
```

Comment:
iirc, `skopeo` (invoked by aklite-apps`) doesn't inject layers that already exists in the docker store. If so, then there  shouldn't be much difference between `aklite-apps run` and `docker compose up` if images/apps are preloaded to the docker store.